wiki/Installation-guide.md

```diff
@@ -96,11 +96,39 @@ If you do not install the correct version of FFmpeg systemwide, a common problem
 * Select `pipewire` or `pulse` as the device.
 * Connect with headset and wait until streaming starts.
 * In `pavucontrol` set the device ALVR is recording from to "Monitor of \<your audio output\>". You might have to set "Show:" to "All Streams" for it to show up.
-* Any audio should now be played on the headset, optionally you can mute the audio output.
+* Any audio should now be played on the headset. To automatically mute your PC speakers when the headset is streaming, you can use the following script:
+
+  ~~~
+  #!/bin/sh
+  case $ACTION in
+          connect)
+                  pactl set-sink-mute @DEFAULT_SINK@ 1;;
+          disconnect)
+                  pactl set-sink-mute @DEFAULT_SINK@ 0;;
+  esac
+  ~~~
+
+  Save this text to a file, make it executable (`chmod +x ...`) then put the file name in "on connect script" and "on disconnect script" settings (Connection tab).
```

Comment:
Please mention that you need to enable advanced options to see the on connect script and on disconnect script

Reply:
Other than that LGTM!